From e5dced8be1a3f56eca431de28dcc69e3a6768de1 Mon Sep 17 00:00:00 2001
From: Richard Bradley <richard.bradley@joyent.com>
Date: Tue, 7 May 2019 18:39:19 +0100
Subject: [PATCH] MANATEE-380 would like `manatee-adm promote` MANATEE-393
 manatee-adm does not listen for PG errors after connecting MANATEE-417
 Manatee should not use the master branch for its state machine dependency

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 365e500..c9ab264 100644
--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
     },
     "dependencies": {
         "manta": "1.2.6",
-        "manatee": "git+https://github.com/joyent/manatee.git#33b33433070e1c83622ca6a555b4bc38f1eec0e6",
+        "manatee": "git+https://github.com/joyent/manatee.git#6e802583f517e74f8e6c1ccab26b01ce2bbbf8f9",
         "carrier": "0.2.0"
     },
     "sdcDependencies": {
-- 
2.21.0

